benchmarks/image-classification/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM rust:1.54
+
+RUN rustup target add wasm32-wasi
+WORKDIR /usr/src
+ADD rust-benchmark rust-benchmark
+WORKDIR /usr/src/rust-benchmark
+RUN cargo build --release --target=wasm32-wasi
+RUN cp target/wasm32-wasi/release/image-classification-benchmark.wasm /benchmark.wasm
```

Comment:
Yeah, I agree with your comment above about changing the `build.sh` to be a more liberal "copier." The problem is that I'm not sure you can use glob patterns like `*benchmark.wasm` with Docker commands? If you can, then great, we should do that; otherwise we may have to modify `build.sh` to copy everything from some directory, e.g., `/sightglass-output`.

Reply:
I'm actually surprised by this question: how does the "rebuild" CI not fail? It seems like it should because this `Dockerfile` emits a `benchmark.wasm` but you have checked in a `image-classification-benchmark.wasm` &mdash; that should result in a failure when CI realizes we are not building the same files.

I think the answer is that `build-all.sh` picks a random subset of benchmarks to build, so this will eventually fail, just not every time. (Rebuilding the benchmarks takes a while so in an effort to speed up CI we just do a subset). You might want to run `build.sh` on this benchmark and see what happens.

(On a side note, I think at some point there was a Git check to make sure none of the bytes had changed, e.g., `git diff --exit-code`. That is no longer there so let me open an issue...)

Reply:
@abrown Not sure about why the CI didn't fail, but I just made some changes so now build.sh allows for different wasm file names besides benchmark.wasm. Now you can either pass the filename.wasm to build.sh, or it will look in the benchmark you are building for the name and use that.